### PR TITLE
cmd/atlas/internal: add sumfile to remotedir memdir

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: cmd/atlas/internal/cmdext/cmdext_test.go [hunk 1]
```diff
@@ -247,6 +247,13 @@ func TestRemoteDir(t *testing.T) {
 			if err := d.WriteFile("1.sql", []byte("create table t(c int);")); err != nil {
 				http.Error(w, err.Error(), http.StatusInternalServerError)
 			}
+			checksum, err := d.Checksum()
```
Comment (masseelch) [inline cmd/atlas/internal/cmdext/cmdext_test.go:250]:
```suggestion
			sum, err := d.Checksum()
```

File: cmd/atlas/internal/cmdext/cmdext_test.go [hunk 2]
```diff
@@ -247,6 +247,13 @@ func TestRemoteDir(t *testing.T) {
...
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+			}
+			if err := migrate.WriteSumFile(&d, checksum); err != nil {
```
Comment (masseelch) [inline cmd/atlas/internal/cmdext/cmdext_test.go:254]:
```suggestion
			if err := migrate.WriteSumFile(&d, sum); err != nil {
```

File: cmd/atlas/internal/cmdext/cmdext.go [hunk 3]
```diff
@@ -391,7 +392,8 @@ func memdir(client *cloudapi.Client, dirName string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	md := migrate.OpenMemDir(dirName)
+	mdn := "mem://" + dirName
+	md := migrate.OpenMemDir(mdn)
```
Comment (a8m) [inline cmd/atlas/internal/cmdext/cmdext.go:396]: You should open it without the prefix (mem://).

Comment (a8m) [reply cmd/atlas/internal/cmdext/cmdext.go:396]: See: https://github.com/ariga/atlas/blob/master/cmd/atlas/internal/cmdapi/migrate.go#L1347

**Review by masseelch — APPROVED** [review]

File: cmd/atlas/internal/cmdext/cmdext.go (hunk 3 above)
Comment (rotemtam) [reply cmd/atlas/internal/cmdext/cmdext.go]: done

**Review by a8m — APPROVED** [review]

**Pull request merged**